### PR TITLE
resume CloudControllerManagerPort to hardcode in port.go

### PR DESCRIPTION
**Pull request by JornShen**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

resume `CloudControllerManagerPort` to hardcode 10258 for easy reading.

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/cluster/ports/ports.go [hunk 1]
```diff
@@ -43,7 +39,4 @@ const (
 	// KubeControllerManagerPort is the default port for the controller manager status server.
 	// May be overridden by a flag at startup.
 	KubeControllerManagerPort = 10257
-	// CloudControllerManagerPort is the default port for the cloud controller manager server.
-	// This value may be overridden by a flag at startup.
-	CloudControllerManagerPort = cloudprovider.CloudControllerManagerPort
```
Comment (lavalamp) [inline pkg/cluster/ports/ports.go:48]: Please don't delete this, but you may copy-paste the value to remove the package dependency and/or improve the comment. This file is the closest thing we have to seeing all of our port defaults in one place, so it's important documentation.

Comment (andrewsykim) [reply pkg/cluster/ports/ports.go:48]: +1 -- we should just hardcode this to `10258` and leave a comment at the top of this file to not remove any of the reserved ports here.

Comment (JornShen) [reply pkg/cluster/ports/ports.go:48]: sorry for that. updated it. @lavalamp @andrewsykim

**Pull request merged**